bin/generate_driver

```diff
@@ -139,14 +165,36 @@ end
 file "spec/#{snake_name}_spec.rb" do
   content <<-EOS
 describe "Chef::Provisioning::#{camel_name}" do
-  extend #{camel_name}Support
-  # include #{camel_name}Config   # uncomment to get `chef_config` or to mix in code.
-
-  when_the_chef_12_server "exists", server_scope: :context, port: 8900..9000 do
-    with_#{snake_name} "integration tests" do
-      context "machine resource" do
-        it "doesn't run any tests" do
-        end
+  include_context "run with driver", :driver_string => "#{snake_name}"
+
+  with_chef_server do
```

Comment:
Nope, `with_chef_server` actually calls `when_the_chef_12_server`, but

1. with a clearer name, and
1. with default values.

All the `chef-provisioning-aws` invocations look like

    when_the_chef_12_server "exists", organization: 'foo', server_scope: :context

and this encapsulates (overridable) defaults

    organization: "spec_tests", server_scope: :context, port: 8900..9000